pegged/peg.d

```diff
@@ -28,7 +28,7 @@ import std.typetuple;
 @safe:
 
 // Returns quoted and escaped version of the input, but if the input is null, then return `"end of input"`.
-package string stringified(string inp) pure
+package string stringified(scope string inp) pure
```

Comment:
Shouldn't this just be `package string stringified(return string inp) pure`? Then I would expect the change below to be unnecessary.

https://dlang.org/spec/function.html#return-ref-parameters

I'm quite new to dip1000 though, so I could well be wrong.

Reply:
It shouldn't be that, because format allocates a new array as well, same situation of .array() here, if you take a look at the implementation it uses an appender as a writter on the formatter.